cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
Yea, I think this is reasonable. If someone comes with an error without context, we can always ask them to run the command again with `-d` I suppose